chef/roles/dns-client.rb

```diff
@@ -2,7 +2,7 @@
 name "dns-client"
 description "DNS Client Role - Configures the resolver to point at the DNS server"
 run_list(
-         "recipe[resolver]"
+         "recipe[dns::role_dns_client]"
```

Comment:
Style/FirstParameterIndentation: Indent the first parameter one step more than the start of the previous line.